### PR TITLE
avoid ctime race conditions in reverse signatures

### PR DESCRIPTION
**Pull request by oconnor663**
r? @maxtaco @mlsteele

### CODE REVIEW

File: go/engine/device_keygen.go [hunk 1]
```diff
@@ -168,7 +168,7 @@ func (e *DeviceKeygen) Push(ctx *Context, pargs *DeviceKeygenPushArgs) error {
 
 	ds = e.appendEncKey(ds, ctx, encSigner, eldestKID, pargs.User)
 
-	var pukSigProducer libkb.AggSigProducer = nil
+	var pukSigProducer libkb.AggSigProducer // = nil
```
Comment (oconnor663) [inline go/engine/device_keygen.go]: linter complained about this line

Comment (mlsteele) [inline go/engine/device_keygen.go]: bah sorry. The pre-commit linter? It would be great to run these in ci.

Comment (oconnor663) [inline go/engine/device_keygen.go]: Yeah, though sometimes the lints just change, and it hits the next person who edits the file :p

**Comment by oconnor663** [comment]
Compare units here: https://github.com/keybase/client/blob/00e7c0723638c2af9d760c866c6f12ea9559b30e/go/libkb/kbsig.go#L289-L292

**Comment by mlsteele** [comment]
Cool, thanks

**Pull request merged**